src/routes/assessment.ts

```diff
@@ -33,7 +33,7 @@ export async function handleGetAssessmentRecommendations(c: ContextWithAuth) {
       return c.json({ error: "Current module is required" }, 400);
     }
 
-    const assessmentService = new AssessmentService(c.env.DB);
+    const assessmentService = getAssessmentService(c.env);
```

Comment:
fucking c.env fills me with mrage

Reply:
... is this something I need to change? lol